README.md

```diff
@@ -36,9 +59,27 @@ verbing('swimming')
 verbing('go')
 // =>  'go'
 ```
- 
+//Answer to 6 lines 63-74-??? Not clear
+let str = "dock"
+let atLeastThree= "ing"
+let alreadyEnding = "ly"
+
+if (str.length >= 3){
+  console.log(str + atLeastThree)
+} else {
+if (str + atLeastThree)
+  console.log(str + "ly")
+if str.length < 3,
+  console.log(str)
+}
+
+
 7. Create a `withoutLast` code block that takes a single string variable, and returns a copy of the string without the last word.
 
+let words = "Let them eat cake";
```

Comment:
Same hard coding issue. Would this code work if we change the words to "I'm happy to be learning Javascript" ? 
Working solution would look like this: 

let h = "I'm happy to be learning JavaScript";
let words = h.split(" ");
let withOutLast = words.slice(0, words.length - 1);
let combined = withOutLast.join(" ");